pkg/runner/config.go

```diff
@@ -0,0 +1,99 @@
+package runner
```

Comment:
It's a little bit confusing to put these files under `pkg/runner`. Maybe `pkg/test/runner` or something.

Reply:
moved to pkg/test/runner